docker/router/ssl/localhost.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
The certificate will expire on April 22 and these tests will begin to fail.
let's change it to something that expires somewhere in 1000 years

```
> openssl x509 -text -noout -in foo
Certificate:
...
Validity
...
            Not After : Apr 22 18:58:31 2024 GMT
```

Reply:
Now its will expire in Jul 27 09:18:42 3023 GMT